contracts/verification/OraclizeVerifier.sol

```diff
@@ -4,22 +4,26 @@ import "../Manager.sol";
 import "./IVerifier.sol";
 import "./IVerifiable.sol";
 
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
 import "../../installed_contracts/oraclize/contracts/usingOraclize.sol";
 
 
 /*
  * @title Verifier contract that uses Oraclize for off-chain computation
  */
 contract OraclizeVerifier is Manager, usingOraclize, IVerifier {
+    using SafeMath for uint256;
+
     string public verificationCodeHash;
+    uint256 public gasPrice;
```

Comment:
Since the price of using Oraclize includes the price of the callback, we can set the gas price and gas limit to be used for the callback. Oraclize does not provide refunds if the gas provided is not all used, so it is important to accurately estimate the cost of the callback...We can probably do this and then set the gasLimit and gasPrice to be used in the constructor, but perhaps we want to introduce setters to change the values used over time - less so for gasLimit, more so for gasPrice.